clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClientBuilder.java

```diff
@@ -154,6 +154,12 @@ ZeebeClientBuilder jobWorkerExecutor(
    */
   ZeebeClientBuilder maxMessageSize(int maxSize);
 
+  /**
+   * A custom streamEnabled allows the client to use job stream instead of job poll. The default
+   * value is set as enabled.
+   */
+  ZeebeClientBuilder streamEnabled(boolean streamEnabled);
```

Comment:
:x: Let's make it clear this is a default and will be overridden by the worker specific setting, by calling it defaultJobWorkerStreamEnabled :+1: